draft-ietf-taps-interface.md

```diff
@@ -1521,14 +1528,14 @@ Type:
 : Integer
 
 Default:
-: infinite
+: infinite (a special value, e.g. -1)
```

Comment:
Do we need to specify an integer value like this at all?

Reply:
Ah, I agree, changing this to "Numeric" is a better solution!